support messenger 5.0

yuripave opened the pull request:
Hello, appreciate the efforts on phpgears packages. We are using the cqrs package with symfony messenger and would like the integration to support messenger 5.0.

See [changelog](https://github.com/symfony/messenger/blob/5.0/CHANGELOG.md) for both 4.4 and 5.0, it did introducing some breaking changes but this package is not affected in any way. Thanks!

juliangut commented:
Firstly thanks for the cheering

As you can see Travis build is failing. That build is trying to install minimum version of dependencies and gets stalled, it seems due to composer not resolving dependencies in a limited time-frame (10 minutes).

It might help updating Travis build to use php 7.4 (without "snapshot") at https://github.com/phpgears/cqrs-symfony-messenger/blob/master/.travis.yml#L18

yuripave committed:
support messenger 5.0

yuripave committed:
test on php7.4

yuripave commented:
The build was actually failing on the [php7.1](https://travis-ci.org/phpgears/cqrs-symfony-messenger/jobs/659106723). I now changed to only add `^5.0` meaning minimum still support `^4.3`. Somehow updating to `^4.4` hit a [composer bug](https://github.com/composer/composer/issues/7051) when update with `prefer-lowest`. 

Nevertheless I added a commit to update travis to test with `php7.4`, as I see you were doing the same upgrade on other repos recently.
